test/test_parameter_map.py

```diff
@@ -186,8 +189,11 @@ def test_parameter_set_opt() -> None:
     """
     Test the setting in optimizer format.
     """
-    amp_ids = [[("RX90p", "d1", "gauss", "amp")], [("RX90m", "d1", "gauss", "amp")]]
+    amp_ids = [
+        [("RX90p[0]", "d1", "gauss", "amp")],
+        [("RX90m[0]", "d1", "gauss", "amp")],
+    ]
     pmap.set_opt_map(amp_ids)
     pmap.set_parameters_scaled([-1.0, 1.0])  # -+1 correspond to min and max allowd
-    assert pmap.get_parameter(("RX90p", "d1", "gauss", "amp")).get_value() == 0.4
-    assert pmap.get_parameter(("RX90m", "d1", "gauss", "amp")).get_value() == 0.6
+    assert pmap.get_parameter(("RX90p[0]", "d1", "gauss", "amp")).get_value() == 0.4
```

Comment:
What is the difference between `RX90p` and `rx90p`?

Reply:
As discussed with @nwittler, we make everything in small letter.

Reply:
Yes, I'll change it to comply with the new naming scheme.